<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSettingDialog</class>
 <widget class="QWidget" name="QtSettingDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>461</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="text">
     <string>地址</string>
    </property>
    <property name="buddy">
     <cstring>rtspUrlEdit</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="rtspUrlEdit"/>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>网络</string>
   </property>
   <widget class="QRadioButton" name="rbUdpTransport">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbTcpTransport">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>渲染方式</string>
   </property>
   <widget class="QRadioButton" name="rbImageReader">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGdiRender">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GDI</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>QtSettingDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
